docs/index.rst

```diff
@@ -252,6 +252,41 @@ Support for a growing number of locales (see ``locales.py`` for supported langua
     >>> future.humanize(a, locale='ru')
     'через 2 час(а,ов)'
 
+Dehumanize
+~~~~~~~~
+
+Dehumanize a string representing a past time:
```

Comment:
How about `"Take a human readable string and use it to shift into a past time."`